modules/sum-transformers/additional/summary/summary_result.go

```diff
@@ -0,0 +1,87 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package summary
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/semi-technologies/weaviate/entities/models"
+	"github.com/semi-technologies/weaviate/entities/search"
+	"github.com/semi-technologies/weaviate/modules/sum-transformers/ent"
+)
+
+func (p *SummaryProvider) findSummary(ctx context.Context,
+	in []search.Result, params *Params) ([]search.Result, error) {
+	if len(in) == 0 {
+		return nil, nil
+	} else if len(in) > 0 {
```

Comment:
I think this part can be just `else` bc if it's not equal to `0` then it is for sure `> 0`

Reply:
Done